### PR TITLE
NO-JIRA Update manager-run.sh to parametrize the MC control plane URL

### PR DESCRIPTION
**Pull request by lucamolteni**
No Jira

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.

</details>

### CODE REVIEW

File: dev/bin/manager-run.sh [hunk 1]
```diff
@@ -48,7 +48,7 @@ mvn \
   -Dmanaged-connectors.kafka.client.id=${mc_client_id} \
   -Dmanaged-connectors.kafka.client.secret=${mc_client_secret} \
   -Dmanaged-connectors.kafka.security.protocol=SASL_SSL \
-  -Dmanaged-connectors.services.url=https://cos-fleet-manager-cos.rh-fuse-153f1de160110098c1928a6c05e19444-0000.eu-de.containers.appdomain.cloud \
```
Comment (r00ta) [inline dev/bin/manager-run.sh:51]: If this is a new parameter required by the script could you update the header of the file 
```
########
# Run Fleet Manager locally in dev mode
#
# Env vars:
```

and the dev/README.md?

File: dev/bin/manager-run.sh [hunk 2]
```diff
@@ -48,7 +48,7 @@ mvn \
...
+  -Dmanaged-connectors.services.url=${MANAGED_CONNECTORS_CONTROL_PLANE_URL} \
```
Comment (manstis) [inline dev/bin/manager-run.sh:51]: Erm.. is this going to run OK locally now? Should the environment variable not be set[*] somewhere too?

[*] Set to a meaningful value.

Comment (r00ta) [reply dev/bin/manager-run.sh:51]: if this is a new parameter needed by the script could you update the header of the file 
```
# Run Fleet Manager locally in dev mode
#
# Env vars:
```
and `dev/README.md`?

**Review by Srihari1192 — APPROVED** [review]

File: dev/bin/manager-run.sh (hunk 2 above)
Comment (jomarko) [reply dev/bin/manager-run.sh:51]: +1

File: dev/README.md [hunk 3]
```diff
@@ -36,9 +36,10 @@ Here is a list of the **required** environment variable:
...
 |:--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|
-| `MANAGED_CONNECTORS_CLUSTER_ID` | ID of the cluster where Managed Connectors are deployed, required to use Managed Connectors based actions. Skip it if you need them. |
 | `MANAGED_KAFKA_INSTANCE_NAME`   | Name of the remote Managed Kafka instance, used to configure the instance itself and the related service accounts.                  |
 | `OPENSHIFT_OFFLINE_TOKEN`       | OpenShift offline token. To obtain it, go to https://console.redhat.com/openshift/token                                             |
+| `MANAGED_CONNECTORS_CLUSTER_ID` | ID of the cluster where Managed Connectors are deployed, required to use Managed Connectors based actions. Skip it if you don't need MC actions. |
+| `MANAGED_CONNECTORS_CONTROL_PLANE_URL` | ID of the MC control plane, required to use Managed Connectors based actions. Skip it if you don't need MC actions. |
```
Comment (jomarko) [inline dev/README.md:42]: is it ID or URL ?

**Review by jomarko — COMMENTED** [review]
Thank you @lucamolteni, one question

**Review by jomarko — COMMENTED** [review]
@lucamolteni thank you for PR, please have a look on my comment https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/548/files#r838480659 and I think I can approve then.

**Review by manstis — APPROVED** [review]
Approving, but @jomarko comment should be addressed.

**Comment by lucamolteni** [comment]
> @lucamolteni thank you for PR, please have a look on my comment https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/548/files#r838480659 and I think I can approve then.

Thank you please reveiw again

**Review by jomarko — APPROVED** [review]
Thank you

**Pull request merged**